pkg/ipam/metadata/manager.go

```diff
@@ -7,6 +7,8 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/cilium/cilium/pkg/ipam"
```

Comment:
nit: This import here is sorted in the wrong order, this should be part of the local package group below.

Reply:
Sure, thank you for pointing that out.